components/Wallet/PlaceOrder/Form/TradeInputs.jsx

```diff
@@ -155,7 +155,7 @@ export const TradeInputs = ({
         }}
         value={order.price || ''}
         onChange={handleChange}
-        placeholder='0.00'
+        placeholder={`0.${'0'.repeat(Math.max(0, 6 - 4))}`}
```

Comment:
This one can just be hard-coded to 0.00 Since it is always the same

Reply:
Yes, we can change it back when we do inversion.